acbuild/acbuild.go

```diff
@@ -283,19 +317,22 @@ func stdout(format string, a ...interface{}) {
 }
 
 func addACBuildAnnotation(cmd *cobra.Command, args []string) error {
-	const annoNamePattern = "appc.io/acbuild/command-%d"
+	const annoNamePattern = "coreos.com/acbuild/command-%d"
```

Comment:
I changed it since acbuild doesn't only produce AppC images anymore. I can change it back if you think leaving `appc.io` annotations on OCI images would be fine.